src/Raven.Server/Documents/ETL/Providers/AI/Handlers/AiEtlConnectionHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Threading.Tasks;
+using Raven.Server.Documents.ETL.Providers.AI.Handlers.Processors;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Documents.ETL.Providers.AI.Handlers;
+
+public sealed class AiEtlConnectionHandler : DatabaseRequestHandler
+{
+    [RavenAction("/databases/*/admin/etl/ai/azureopenai/test-connection", "POST", AuthorizationStatus.DatabaseAdmin)]
```

Comment:
maybe let's simplify this
send `type=AiConnectorType` as a parameter and the deserialize it correctly, then we need only one endpoint

Reply:
Done